.github/workflows/raise-pr-for-constants.yml

```diff
@@ -0,0 +1,81 @@
+name: Raise PR for Destination Constants
+
+on:
+  push:
+    branches:
+      - develop
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  BRANCH_NAME: chore/gh-update-destinations-ts
```

Comment:
It's not a chore if the changes include updating/deleting some constants. Let's use `fix` in the branch name and commit type so it gets highlighted in the release notes as well.

Reply:
Updated as per our discussion